app/models/mixins/live_metrics_mixin.rb

```diff
@@ -1,8 +1,13 @@
 module LiveMetricsMixin
   extend ActiveSupport::Concern
 
+  LIVE_METRICS_DIR = Rails.root.join("product/live_metrics")
+
   class MetricValidationError < RuntimeError; end
 
+  delegate :metrics_available, :to => :metrics_capture
```

Comment:
why is the delegate needed here? (and one line below too)
also, afaik when using `delegate`, the "to" part holds the name of the class that it should delegate to, not the method name.
perhaps `alias_method` is the way to go here.